theano/__init__.py

```diff
@@ -67,7 +67,7 @@
     predefined_modes, predefined_linkers, predefined_optimizers,
     FunctionMaker, function, function_dump, OpFromGraph,
     ProfileMode, ProfileStats,
-    Param, shared, as_op)
```

Comment:
Actually, this is breaking something in the deep learning tutorials.
Should we keep an alias `Param = In`, or a dummy `Param` class that calls `In` with a deprecation warning?